### PR TITLE
Milestone 1: Setup project

### PR DESCRIPTION
**Pull request by D4nielJ**
## Hello reviewer :vulcan_salute: 

This PR adds, according to the requirements, the following:
* Project setup with webpack
* Correct gitflow
* Basic HTML structure
* Basic JS structure using ES2016 modules.

Thank you :partying_face: 

P.S.: [Live demo here](https://deploy-preview-1--zealous-heyrovsky-c6c3cc.netlify.app/?utm_source=github&utm_campaign=bot_dp)

### CODE REVIEW

File: src/app/run.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+   */
+  let loadedData = [];
+  loadedData = fetchData(data);
+
+  // 2. Update DOM
+  updateDOM(loadedData);
+
+  // TODO: 3. submit info from the form -> update DOM;
+  // TODO: 4. Refresh button -> update DOM;
+}
```
Comment (dasileker) [inline src/app/run.js]: - [x] Please remove commented lines and use **`arrow function`** instead of **`function`**.

File: src/app/fetchData.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default function fetchData(data = []) {
+  const loadedData = data;
+  return loadedData;
+}
```
Comment (dasileker) [inline src/app/fetchData.js]: - [x] Use best practice for ES6.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @D4nielJ**, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight
- Good use of webpack.
- Nice touch on deploying the app on an external host.
- Nice README file.

## Required Changes ♻️
**_Check the comments under the review 👇._**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/app/fetchData.js (hunk 2 above)
Comment (Davidosky007) [reply src/app/fetchData.js]: * done 👍

File: src/app/run.js (hunk 1 above)
Comment (Davidosky007) [reply src/app/run.js]: * done 👍

**Comment by Davidosky007** [comment]
# Hi @D4nielJ 👋, 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏
# [optional]
* Please note that for javascript projects the right linter to use is [Javascript linters](https://github.com/microverseinc/linters-config/tree/master/javascript) and not [HTML + CSS + JS](https://github.com/microverseinc/linters-config/tree/master/html-css-js) 
* The HTML+CSS+JS is made for the Portfolio project which needs Javascript for interactions as a result the linters was made for easy use 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**